scripts/packaging/debian-control/elektra-bin/postinst

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 set -e
 
-dpkg-maintscript-helper rm_conffile /etc/bash_completion.d/kdb 0.8.11-1~ -- "$@"
\ No newline at end of file
+dpkg-maintscript-helper rm_conffile /etc/bash_completion.d/kdb 0.8.11-1~ -- "$@"
```

Comment:
What is this script doing? Why is a very outdated version number here?

The bash completion is not working well (or even not at all?), if there are packaging problems with it, it is probably better to simply remove it completely.

Reply:
This is the generated output of the [elektra-bin.maintscript](https://github.com/ElektraInitiative/libelektra/blob/debian/debian/elektra-bin.maintscript). 

The purpose of it was to remove a conffile which was necessary for versions prior to 0.8.11-1. 
Since 0.8.11-1 was already 5 years ago, I think these maintscripts can be removed.